webfront/views/modifiers.py

```diff
@@ -853,5 +853,15 @@ def get_model_structure(value, general_handler):
     return get_model_structure
 
 
+def get_subfamilies(value, general_handler):
+    queryset = general_handler.queryset_manager.get_queryset().first()
+    entries = Entry.objects.filter(integrated=queryset.accession, source_database='panther', is_alive=False)
```

Comment:
We should not filter by `source_database` as this modifier will be used for FunFam.

Reply:
Change made. I also added a fixture for funfams and updated the tests, please check 1fdbf66